### PR TITLE
Update aarch64-kasan.adoc

### PR DESCRIPTION
**Pull request by grahamperrin**
Grammar: 

> … As of … is …

A full stop for a final list item. 

bhyve, lowercase. <https://man.freebsd.org/cgi/man.cgi?query=bhyve&sektion=8&manpath=freebsd#DESCRIPTION>

### CODE REVIEW

**Comment by grahamperrin** [comment]
Also, there's no `Links:` section.

File: website/content/en/status/report-2023-01-2023-03/aarch64-kasan.adoc [hunk 1]
```diff
@@ -8,14 +8,14 @@ They are particularly effective in combination with regression testing suites or
...
-As of gitref:89c52f9d59fa[repository=src], the kernel address sanitizer that was previously exclusive to amd64 has now been ported to arm64.
+As of gitref:89c52f9d59fa[repository=src], the kernel address sanitizer that was previously exclusive to amd64 is ported to arm64.
 
 Prior testing has been done on a decent variety of machines, including:
 
 - Various Ampere Altra machines
 - QEMU
 - Microsoft's "Volterra" Devkit
-- BHyve (WIP)
+- bhyve (WIP).
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-01-2023-03/aarch64-kasan.adoc:18]: I do not remember how we dealt with punctuation in lists last quarter. I prefer to leave status reports authors do what they prefer about it (I personally add a `;` at the end of each line element, except last one which gets a `.` if the sentence is finished).

If you want to add the `.`, I am not against, but please ensure at least that the listing style is consistent with whatever we did last quarter.

**Review by lsalvadore — APPROVED** [review]
@kevans91: As @grahamperrin noted, there is no link section in your report. You do not need to have one, but if you have any general link that could fit in that section, I would recommend to add it.

File: website/content/en/status/report-2023-01-2023-03/aarch64-kasan.adoc (hunk 1 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-01-2023-03/aarch64-kasan.adoc:18]: Thanks. I sped through history for last quarter, found https://github.com/freebsd/freebsd-quarterly/commit/76535b3ef58c2ce63a8ee21b9267ccf035179602 with: 

>> * where a list is preceded by a sentence with a colon, close the list with a full stop.

**Comment by lsalvadore** [comment]
This has been merged by @grahamperrin with commit https://github.com/freebsd/freebsd-doc/commit/74f28f7379a4d1b637252de112ff93780423206c and thus it can be safely closed.

**Pull request closed without merging**